src/main/java/core/basesyntax/action/ActionHandler.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.action;
+
+public interface ActionHandler {
+    int performAction(int amount);
```

Comment:
all ActionHandlers are invalid - they don't work. please, pay more attention to these moments before next submitting
```suggestion
    void handleTransaction(FruitTransaction transaction);
```